apps/src/templates/teacherNavigation/TeacherNavigationBar.tsx

```diff
@@ -73,8 +75,26 @@ const TeacherNavigationBar: React.FunctionComponent = () => {
   }
 
   const performanceSectionTitle = getSectionHeader(i18n.performance());
+
+  if (
```

Comment:
This should be in a useEffect

Reply:
Also, should this logic be in the navigation bar or somewhere else, maybe in the `TutorTab` itself?